packages/next-playwright/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "sourceMap": true,
+    "esModuleInterop": true,
+    "target": "es2015",
+    "downlevelIteration": true,
+    "preserveWatchOutput": true,
+    "outDir": "dist"
```

Comment:
At a minimum I'd recommend enabling `strict` here (which includes a few things).

We use `noUncheckedIndexedAccess` in a lot of places now too.

Reply:
Ok. I got an impression that this was the default template for Next.js and wanted to stay close to it. I'll check with them on this.